### DISCUSSION TITLE
How to customize some parameters

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#11599] [Q&A] [answered]
How to write agent.instance_properties_json={"systemId":"qazwsx123456"} to data_binary; or add custom parameters to data_binary through -javaagent;Or use some other methods to achieve; I need to log some custom parameters

Example:

**systemId: "qazwsx123456"**
traceId: "15ee94eb73dd487ab3471b9d064ebd46.114.17010537814230001"
traceSegmentId: "15ee94eb73dd487ab3471b9d064ebd46.114.17010537814230000"
spans {
  spanId: 1
  startTime: 1701053781424
  endTime: 1701053781424
  operationName: "Druid/Connection/getConnection"
  spanType: Local
  componentId: 115
}
spans {
  parentSpanId: -1
  startTime: 1701053781423
  endTime: 1701053781431
  operationName: "GET:/api/query"
  spanLayer: Http
  componentId: 14
  tags {
    key: "url"
    value: "http://127.0.0.1:8888/api/query"
  }
  tags {
    key: "http.method"
    value: "GET"
  }
  tags {
    key: "http.status_code"
    value: "200"
  }
}
service: "direct-web6"
serviceInstance: "1c193fdefc5d42dbbca62981ac795498@172.16.23.1"

**Comment by wu-sheng** [comment, marked as answer]
I can't see the point. You could add any tags you want from the toolkit manually. But, it would be a configuration from agent side. We can't see the meaning of adding a static config to all traces. But, you could do on your own.

**Reply by mitht** [reply]
I have a lot of microservices. Now I only know the service name. I need to know which system they are.

**Reply by wu-sheng** [reply]
I can't see the point still. The name should be unique.

**Reply by mitht** [reply]
A system has multiple microservices, and each service is automatically monitored through -javaagent. When I have multiple systems, I need to use systemId to distinguish which system these services belong to.
How can I achieve this function through static config;

**Reply by wu-sheng** [reply]
The name is given by yourself, not us. You could use things like group names or suffixes to distinct. But that is on you. SkyWalking doesn't control your management process.
